### PR TITLE
OZ-468: Add docs to create own distribution.

### PR DESCRIPTION
**Pull request by rbuisson**
Mostly documents how to use the Ozone Maven Archetype.
Also start/stop/destroy instructions.

https://github.com/ozone-his/ozone-docs/assets/10692811/e6489fff-672f-4348-8de6-9583eace7a33

Mostly reuses @ibacher 's PR, but with some adjustments: https://github.com/ozone-his/ozone-docs/pull/1

### CODE REVIEW

**Review by ibacher — APPROVED** [review]
LGTM

File: docs/create-distro.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+## Available commands
+
+|Action|Command|
+|:----|:----|
+|Build the distribution|<pre>./scripts/mvnw clean package</pre>|
+|Run the distribution|<pre>source target/go-to-scripts-dir.sh<br/><br/>./start-demo.sh</pre>|
+|Stop the distribution|<pre>./stop-demo.sh</pre>|
+|Destroy the distribution|<pre>./destroy-demo.sh</pre>|
```
Comment (mks-d) [inline docs/create-distro.md:90]: I'll be merging this for now, but I would like to hear from you (@ibacher, @rbuisson) about this table presentation here.

**Review by mks-d — APPROVED** [review]

File: docs/create-distro.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  |    |    |
+  |    |    |--frontend_config
+  |    |    |--initializer_config
+  |--readme
+  |    |--impl-guide.md
+  |--scripts
+  |--.gitignore
+  |--.gitpod.yml
+  |--pom.xml
+  |--README.md
```
[the remark this answers is not in the document]
Comment (ibacher) [reply docs/create-distro.md]: I don't think we're creating a local Git repository. I didn't want to make assumptions about having anything other than Maven installed.

File: docs/create-distro.md (hunk 1 above)
Comment (ibacher) [reply docs/create-distro.md:90]: Seems right. The last two commands do assume you're in the directory the `./start-demo.sh` script is launched from, but I don't know how to make that clear.

**Pull request merged**